packages/local_contexts/local_contexts.js

```diff
@@ -0,0 +1,169 @@
+var cvoc_lc_projectSelector = "span[data-cvoc-protocol='localcontexts']";
+var cvoc_lc_projectInputSelector = "input[data-cvoc-protocol='localcontexts']";
+
+var cvoc_lc_lcBaseUrl = "https://localcontextshub.org";
+
+var cvoc_lc_seach_minimumInputLength = 4;
+var cvoc_lc_search_delay = 1000;
+
+// if (typeof initFunction !== "function") {
+//     function initFunction() {
+//         console.log("doc ready")
+//         cvoc_lc_viewProject();
+//         cvoc_lc_editProject();
+//     }
+//
+//     $(document).ready(initFunction);
+// }
+
+$(document).ready(() => {
+    // console.log("doc ready")
+    cvoc_lc_viewProject();
+    cvoc_lc_editProject();
+});
+
+function cvoc_lc_buildLCProjectPopup(project) {
+
+    const createItemImage = (notice_label) => {
+        return `<div style="flex;flex-direction: column;text-align: center; max-width: 175px">
+                    <div style="margin: 12px;">
+                        <img loading="lazy" width="119px" src="${notice_label.img_url}" alt="">
+                        <p style="margin-top:8px; font-size: 14px">${notice_label.name}</p>
+                    </div>
+                </div>`
+    }
+
+    // console.log(project)
+    const notices = project.notice || []
+    const labels = (project.tk_labels || []).concat(project.bc_labels || [])
+    const lcWrapper = `<div style="max-width: 700px; margin-top: 10px;
+                border: 0.5px solid darkgray;border-radius: 2rem; background: white">
+        <div  style="padding: 12px;padding-bottom: 0; width: 100&; display: flex; flex-wrap: wrap; justify-content: flex-start">
```

Comment:
Might be better to add a class and css file. For now though, it looks like the width is incorrect - was it meant to be 100%?